CMSgov/mint-app
# [EASI-3563] - Participant updates

# EASI-3563

## Changes and Description

- Added enum and translation for`ACCOUNTABLE_CARE_ORGANIZATION`
- Added a migration for new value
- Renamed 2 other translations



## How to test this change

Bring up the application

Navigate to the participants and providers page

Confirm the options are added / updated correctly
a. Accountable Care Organization (ACO
b. Community-based organizations (CBO)
c. Legal Entities
d. Medicare providers/suppliers

Select all of the changes values and verify the mutation is successful

View the data in read only view as well to ensure it is as expected

If desired, the new fields can also be tested with the updated postman collection.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
